sklearn_numba_dpex/kmeans/tests/test_kmeans.py

```diff
@@ -212,104 +210,24 @@ def test_relocate_empty_clusters(dtype):
         kmeans_engine.fit(X)
 
     expected_n_iter = 1
-    expected_labels = [0, 0, 0, 0, 0, 0, 0, 2, 2, 1]
+    expected_labels = np.array([0, 0, 0, 0, 0, 0, 0, 2, 2, 1], dtype=np.int32)
+    vanilla_labels = kmeans_vanilla.labels_
+    engine_labels = asnumpy(kmeans_engine.labels_).astype(np.int32)
     assert kmeans_vanilla.n_iter_ == expected_n_iter
     assert kmeans_engine.n_iter_ == expected_n_iter
-    assert_array_equal(kmeans_vanilla.labels_, asnumpy(kmeans_engine.labels_))
-    assert_array_equal(kmeans_vanilla.labels_, expected_labels)
+    assert _is_same_clustering(
+        vanilla_labels,
+        engine_labels,
+        n_clusters=3,
+    )
+    assert _is_same_clustering(vanilla_labels, expected_labels, n_clusters=3)
     assert_allclose(
-        kmeans_vanilla.cluster_centers_, asnumpy(kmeans_engine.cluster_centers_)
+        kmeans_vanilla.cluster_centers_[vanilla_labels],
+        asnumpy(kmeans_engine.cluster_centers_)[engine_labels],
     )
     assert_allclose(kmeans_vanilla.inertia_, kmeans_engine.inertia_)
 
 
-@pytest.mark.parametrize("dtype", float_dtype_params)
-def test_select_samples_far_from_centroid_kernel(dtype):
```

Comment:
Why was this test removed?

Reply:
it's a mistake I think ? i thought it tests the function that has been replaced by our topk :facepalm: